### ISSUE TITLE
manual: thermal expansion coefficient notation

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#989]
The thermal expansion coefficient $\alpha$ is sometimes referred to as $\beta$. Fix this.

**Comment by spco** [comment]
There are 25 occurrences of alpha, 24 of beta. Choose your favourite?

Relatedly, the equation in line 5580 of manual.tex (Rayleigh number definition) uses $\alpha$ for the thermal diffusivity, elsewhere it's normally $\kappa$ (I think this comes direct from Wikipedia). I haven't checked whether any other occurrences are odd usages. This needs fixing.

**Comment by ian-r-rose** [comment]
In the geophysical literature, $\alpha$ is much more common for thermal expansivity, and $\kappa$ is much more common for thermal diffusivity, so I would advocate for those symbols.

**Comment by gassmoeller** [comment]
I agree with Ian on this.

**Comment by spco** [comment]
So, would it be better to use LaTeX macros to help keep things uniform, and also improve readability of the .tex file? Something along the lines of 

```
\newcommand{\diffusivity}{\kappa}
\newcommand{\therm_expan_coeff}{\alpha}
```
? This would require a fair bit of work, it's not a 10 minute job to extend this to all/most variables and coefficients in the manual.

**Comment by gassmoeller** [comment]
I am not sure this would help, because a large part of the manual is taken over from the descriptions in the source files, and you probably do not want to change all of them as well. Also when not so experienced users will write their documentation text in the cc files they are most likely not aware of the existence of these macros, and we will always have a mixture of `\diffusivity` and `\kappa`. Maybe just change all of the wrong occurences of $\beta$ and $\alpha$ (but keep the ones where they are used correctly). A bit of boring work :-(

**Comment by yinghe616** [comment]
There is also a parameter in the entropy stabilization section called 'alpha',  which means the exponent  in the entropy viscosity stabilization. But maybe give it a longer name specifying the functionality could be better? like entropy_viscosity_exponent?  The default value is 2, and most user doesn't worry about this, however the falling box problem only works if I set the value to be 1, which took me a lot of time to figure out but I still don't know exactly why it works... This may not be relevant to this issue, but  @tjhei @bangerth perhaps you can give it a better explanation?

**Comment by bangerth** [comment]
I don't think I can give a better explanation other than referring to other papers. I've done this in #1010.

Why it requires one particular exponent and doesn't work for another -- that is a good question. I must admit that I don't have a good answer. It may be related to #293, but I don't really know :-(

**Comment by egpuckett** [comment]
I'll take this on.

**Comment by lkellogg** [comment]
I agree with Ian; this is more standard in the geophysical literature.

**Comment by egpuckett** [comment]
OK everyone. In Iris van Zelst's benchmark entry

   4.6 The Burstedde variable viscosity benchmark

she uses \alpha in the  equation for the viscosity

   μ = exp {1 − α [x(1 − x) + y(1 − y) + z(1 − z)]}

whereas as near as I can tell Burstedde [BSA + 13], only uses 

  μ = exp(1 − 4(x(1 − x) + y(1 − y) + z(1 − z))).

Does anyone want to suggest a variable other than \alpha (α) here?

**Comment by egpuckett** [comment]
@lkellogg is now working on this starter project.
